torq.q

```diff
@@ -301,7 +335,7 @@ $[count[req] = count req inter key params;
   0<count req inter key params;
 	.lg.o[`init;"ignoring partial subset of required process parameters found on the command line - reading from file"];
   ()];		 
-
+checkdependency[getdepfile]
```

Comment:
r.e. point above, it would be fine to just get rid of getdepfile and do 

checkdependency[getconfig["config.csv";1]]